config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,8 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider: 'AWS',
+    aws_access_key_id: ENV['AWS_ACCESS_KEY_ID'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.